anonymous-chats-backend/models/Chats/Dto/ChatMessageDTO.cs

```diff
@@ -15,8 +15,4 @@ public class CreateChatMessageDTO
     [MaxLength(500)]
     public string FilteredMessage { get; set; } = string.Empty;
 
-    [Required]
-    [MaxLength(255)]
-    public string CreatedBy { get; set; } // Auditing field
-
```

Comment:
How are people on the front end going to know who said this? Or are you going to populate the "alias" of the user so that it's like [Alias] says "Hi!"

Reply:
Good question, we're removing this because we forgot about the CreatedBy field that's inherited from the base model class. We're just gonna use that to maintain which userId created which ChatMessage object